appeals/api/src/server/endpoints/documents/documents.mapper.js

```diff
@@ -138,7 +138,7 @@ export const mapFoldersLayoutForAppealSection = (sectionName, folders) => {
 	/** @type {Object<string, Object>} **/ const folderLayout = {};
 
 	for (const path of FOLDERS) {
-		if (path.indexOf(sectionName) === 0) {
+		if (path.indexOf(sectionName) === 0 && folders.some((folder) => folder.path === path)) {
```

Comment:
Not sure this is going to work. This to get all folders with path starting with `sectionName`...

Reply:
This was added to support the API returning different folders depending on the appeal type, but it has been removed now anyway